rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/TestBase.java

```diff
@@ -168,7 +169,7 @@ private void clearRecoveryLog() {
     }
 
     String getFirstLRA() {
-        Path lraDir = Paths.get(storeDir.toString(), "ShadowNoFileLockStore", "defaultStore", "StateManager", "BasicAction", "TwoPhaseCoordinator", "LRA");
+        Path lraDir = Paths.get(storeDir.toString(), "ShadowNoFileLockStore", "defaultStore", LongRunningAction.getType());
```

Comment:
just guessing but this was written separately to not include delimiters ('/') in the path I guess so this won't work on windows.

Reply:
That is a very good point. In fact the whole method is assuming the ObjectStore is on a file system (so the test would fail, for example, on JDBC and in-memory stores) . I will raise a separate issue to rewrite the test to use the ObjectStore API to get the log (and to also check we aren't doing something similar elsewhere).

Reply:
I will make sure the JIRA says that we need the ability to run the tests on at lease one other store type.

Reply:
The tracking issue is https://issues.redhat.com/browse/JBTM-3398